common/common.go

```diff
@@ -199,6 +201,21 @@ func SharedVolumesPath(root string) string {
 	return filepath.Join(root, sharedVolumesDir)
 }
 
+// CreateSharedVolumesPath ensures the sharedVolumePath for the pod root passed
+// in exists. It returns the shared volume path or an error.
+func CreateSharedVolumesPath(root string) (string, error) {
+	sharedVolPath := SharedVolumesPath(root)
+
+	if err := os.MkdirAll(sharedVolPath, SharedVolumePerm); err != nil {
+		return "", errwrap.Wrap(errors.New("could not create shared volumes directory"), err)
+	}
+	if err := os.Chmod(sharedVolPath, SharedVolumePerm); err != nil {
```

Comment:
I guess we `Chmod` because the directory might already be present and in this case not catched the `perm` parm in `MkdirAll`? It may seem obvious, but can we leave a small comment for the rationale?